### ISSUE TITLE
Accessibility: Toaster

### ISSUE THREAD

**Issue by gfellerph** [swisspost/common-web-frontend#109]
![image](https://user-images.githubusercontent.com/1659006/170209612-247276e5-2232-46a8-92b9-2cb1b335cc90.png)

Toasts in the CWF Demo are not announced by a screen reader, toasts in the [ngx-toastr library](https://ngx-toastr.vercel.app/) are. The difference is that text in toasts in the CWF are not wrapped in a paragraphs.
